src/lib/firestore/post.ts

```diff
@@ -65,6 +68,7 @@ export const createPost = async (recipient: Recipient) => {
   const newPost: Post = {
     id: `${recipient.recipientGroupId}-${moment().utcOffset(9).format('YYMMDD-hhmmss')}`,
```

Comment:
すいませんが、IDはタイムスタンプだけにしたいです。
（またコレクションを0からリセットさせてください）
```suggestion
    id: `${moment().utcOffset(9).format('YYMMDD-hhmmss')}`,
```